pkg/controller/metrics.go

```diff
@@ -27,46 +27,46 @@ type metrics struct {
 }
 
 const (
-	ACTION_SET_COUNTER_VEC_LABEL_RES         = "resolution"
-	ACTION_SET_COUNTER_VEC_LABEL_RES_SUCCESS = "success"
-	ACTION_SET_COUNTER_VEC_LABEL_RES_FAILURE = "failure"
+	ACTION_SET_COUNTER_VEC_LABEL_RES   = "resolution" //nolint:stylecheck // Already defined in docs
```

Comment:
1. Let's verify if the metrics are enabled by default controller install or through a separate helm flag.
2. If it's through a helm flag, then it would be okay to change the documentation.
3. If it's enabled by default, what is the impact of the change? We should have a plan for the transition - a release note about the breaking change and actual change in a subsequent release.
@viveksinghggits @hairyhum wdyt?

Reply:
Metrics are enables via helm flag `metrics.enabled`, but we don't configure labels via helm, as I understand.
I don't see why we shouldn't change it. The variable name is not an input anywhere and I can only find it in `kanister_prometheus_integration.md` which describes how it works but does not suggest you use it.
Is there some other docs which mention this variable?

Reply:
In `kanister_prometheus_integration.md` there are other 2 variables `ACTION_SET_COUNTER_VEC_LABEL_OP_TYPE` and `ACTION_SET_COUNTER_VEC_LABEL_BLUEPRINT` , there is no occurrence of these 2 in codebase apart from this doc. I am not sure what is the use of those and whether to we should rename it the same way as we are doing it for `ACTION_SET_COUNTER_VEC_LABEL_RES` to `ActionSetCounterVecLabelRes`. It seems inconsistent in doc if we don't do.

Reply:
It seems like `kanister-prometheus-integration.md` is outdated as we don't have `ACTION_SET_COUNTER_VEC_LABEL_OP_TYPE` in the code anymore. 
Maybe we should make it consistent with the code or don't have code in `md` files. We can do that in a separate PR of course.